tomcat/src/main/tomcat-base/conf/catalina.properties

```diff
@@ -131,7 +131,8 @@ junit.jar,junit-*.jar,hamcrest-*.jar,easymock-*.jar,cglib-*.jar,\
 objenesis-*.jar,ant-launcher.jar,\
 cobertura-*.jar,asm-*.jar,dom4j-*.jar,icu4j-*.jar,jaxen-*.jar,jdom-*.jar,\
 jetty-*.jar,oro-*.jar,servlet-api-*.jar,tagsoup-*.jar,xmlParserAPIs-*.jar,\
-xom-*.jar
+xom-*.jar,\
+tomcat-gcp-lib*.jar
```

Comment:
The tomcat-gcp-lib is packaged as a fat jar therefore all the others jars that it depends on, like the datastore client lib, are included in that jar.